src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -145,4 +161,33 @@ public void stop(StopConfiguration configuration) throws AppEngineException {
       }
     }
   }
+
+  @VisibleForTesting
+  void checkAndWarnIgnored(Object valueToIgnore, String propertyName) {
```

Comment:
Minor: I'd put the property name first, like how in other situations usually the name/key is first and the value is second.